src/components/Map/SearchPane/Drawer/DrawerContent/EntityDetails/EntityComponents/EntityEmissionsData/BottomViews/EntityActions/EntityActionsList.tsx

```diff
@@ -7,23 +7,38 @@ interface PageProps {
 
 const EntityActionsList: React.FC<PageProps> = (props) => {
     return (
-        <>
-            {props.actions.length > 0 ?
-                <ul className="ion-margin-top">
-                    {
-                        props.actions.map((action, index) => {
-                            return (
-                                <a href={action.link} target="_blank" style={{ color: '#000', textDecoration: 'none' }}>
-                                    <li>{action.title}</li>
-                                </a>
-                            );
-                        })
-                    }
-                </ul>
-            :
-                <IonText>No actions available</IonText>
-            }
-        </>
+      <>
+        {props.actions.length > 0 ? (
+          <ul className="ion-margin-top" style={{ paddingLeft: 8 }}>
+            {props.actions.map((action, index) => {
+              return (
+                <a
+                  href={action.link}
+                  target="_blank"
+                  style={{
+                    display: "flex",
+                    flexDirection: "row",
+                    textDecoration: "none",
+                    alignItems: "center",
+                  }}
+                >
+                  <img
+                    src={`/assets/icon/actionIcons/${action.type}.png`}
```

Comment:
I'd make this a little more robust - check that the action type is one we know about (add a util function like iconForActionType(action.type)), and return a default generic "Action" icon if not. Just in case the data's broken/out-dated. In particular, live data at the moment doesn't have this field.